SocialCareCaseViewerApi.Tests/V1/Boundary/Request/CreateReferralRequestTests.cs

```diff
@@ -0,0 +1,62 @@
+using System.Collections.Generic;
+using System.Linq;
+using Bogus;
+using FluentAssertions;
+using NUnit.Framework;
+using SocialCareCaseViewerApi.Tests.V1.Helpers;
+using SocialCareCaseViewerApi.V1.Boundary.Requests;
+
+namespace SocialCareCaseViewerApi.Tests.V1.Boundary.Request
+{
+    [TestFixture]
+    public class CreateReferralRequestTests
+    {
+        private Faker _faker;
```

Comment:
This can become (i think)

```
private readonly Faker _faker = new Faker()
```

We can then get rid of the SetUp which is ran before every test

Reply:
Changed in 2f5de34